lib/apiservers/portlayer/restapi/handlers/storage_handlers.go

```diff
@@ -161,7 +161,9 @@ func (handler *StorageHandlersImpl) ListImages(params storage.ListImagesParams)
 
 // WriteImage writes an image to an image store
 func (handler *StorageHandlersImpl) WriteImage(params storage.WriteImageParams) middleware.Responder {
+
 	u, err := util.ImageStoreNameToURL(params.StoreName)
+
```

Comment:
to see who's awake!  ;-)  Seriously, it was some logging I added and removed, but failed to revert to the original file.  Corrected now...thanks!